pegen/pegen.c

```diff
@@ -757,11 +757,13 @@ seq_count_dots(asdl_seq *seq)
     for (int i = 0, l = asdl_seq_LEN(seq); i < l; i++) {
         Token *current_expr = asdl_seq_GET(seq, i);
         assert(current_expr->type == ELLIPSIS || current_expr->type == DOT);
```

Comment:
Suggestion: We can move this assert to a new "default" case. It may help if some compiler complains that we are not using all values in the `switch`.